### PR TITLE
tracker Ignore for 24 hours 

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers this enables ignoring for 24 hours in tracker popups now
also cleans up a bunch of old-school tracker stuff
also using keybase.io for pictures and not the api server

@keybase/react-hackers

### CODE REVIEW

File: shared/actions/tracker.js [hunk 1]
```diff
@@ -165,17 +165,6 @@ export function pushDebugTracker (username: string): (dispatch: Dispatch) => voi
   }
 }
 
-export function onFollowChecked (newFollowCheckedValue: boolean, username: string): Action {
```
Comment (chrisnojima) [inline shared/actions/tracker.js]: we don't have this checkbox anymore

File: shared/actions/tracker.js [hunk 2]
```diff
@@ -228,27 +217,12 @@ export function onUnfollow (username: string): TrackerActionCreator {
   }
 }
 
-function onUserTrackingLoading (username: string): Action {
```
Comment (chrisnojima) [inline shared/actions/tracker.js]: unused

File: shared/actions/tracker.js [hunk 3]
```diff
@@ -228,27 +217,12 @@ export function onUnfollow (username: string): TrackerActionCreator {
...
-    type: Constants.onFollowHelp,
-    payload: {username}
-  }
-}
-
-function trackUser (trackToken: ?string): Promise<boolean> {
+function trackUser (trackToken: ?string, localIgnore: bool): Promise<boolean> {
   const options: TrackOptions = {
-    localOnly: false,
+    localOnly: localIgnore,
```
Comment (chrisnojima) [inline shared/actions/tracker.js]: we need to set both

File: shared/actions/tracker.js [hunk 4]
```diff
@@ -305,28 +283,6 @@ export function onFollow (username: string): (dispatch: Dispatch, getState: () =
   }
 }
 
-export function onMaybeTrack (username: string): (dispatch: Dispatch, getState: () => {tracker: RootTrackerState}) => void {
```
Comment (chrisnojima) [inline shared/actions/tracker.js]: we don't do this anymore either

File: shared/actions/tracker.js [hunk 5]
```diff
@@ -346,7 +300,7 @@ function updateUserInfo (userCard: UserCard, username: string, getState: () => {
         followingCount: userCard.following,
         followsYou: userCard.theyFollowYou,
         bio: userCard.bio,
-        avatar: `${serverURI}/${username}/picture`,
+        avatar: `https://keybase.io/${username}/picture`,
```
Comment (chrisnojima) [inline shared/actions/tracker.js]: we switched serverURI to api.keybase.io which doesn't have the right certs we need so we see warnings in the console now. this fixes that

File: shared/tracker/action.render.desktop.js [hunk 6]
```diff
@@ -12,32 +12,32 @@ export default class ActionRender extends Component {
   props: ActionProps;
 
   render () {
-    const {username, loggedIn} = this.props
+    const {loggedIn} = this.props
 
     if (!loggedIn) {
-      return this.renderLoggedOut(styles, username)
```
Comment (chrisnojima) [inline shared/tracker/action.render.desktop.js]: styles is module-level so we don't need it passed around. username isn't used cause its in the parent closure

File: shared/tracker/action.render.desktop.js [hunk 7]
```diff
@@ -87,49 +87,47 @@ export function calcFooterHeight (loggedIn: boolean): number {
   return loggedIn ? 62 : 151
 }
 
-const styles = {
```
Comment (chrisnojima) [inline shared/tracker/action.render.desktop.js]: using the new eslint friendly style

**Comment by chrisnojima** [comment]
looks like this when it works:

screen shot 2016-04-13 at 6 25 13 pm

**Comment by cjb** [comment]
Out of curiosity, do you get the "You have tracked foo" UI state where everything goes green and there's a close button, or does the popup just close?  (Seems like it should just close.)

**Comment by chrisnojima** [comment]
It goes green and you get a close button. lemme run this flow by coyne before we change it

**Comment by cjb** [comment]
Yeah, seems like the wrong thing, since you haven't tracked them.

**Comment by chrisnojima** [comment]
well you actually have so its a little more complex. if you see the tracker again it will be tracked , but locally and you'll see the whatevz tag

**Comment by MarcoPolo** [comment]
Code lgtm

**Comment by chrisnojima** [comment]
spoke to coyne, we'll add the auto close to this, also renaming whatevz to ignore, boo

**Comment by chrisnojima** [comment]
@cjb added auto close and using ignore instead of whatevz

**Comment by cjb** [comment]
Looks really good, thanks! :thumbsup:

**Pull request merged**